target/linux/ramips/dts/mt7620n_hootoo_ht-tm05.dts

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "hootoo,ht-tm05", "ralink,mt7620n-soc";
+	model = "HooToo HT-TM05";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &ethernet;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "filehub:blue:power";
+			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		wifi {
+			label = "filehub:green:wifi";
+			gpios = <&gpio2 0 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "phy0tpt";
+		};
```

Comment:
@adrianschmutzler LED labels `"filehub:*:*"` here are a preparation of an upcoming harmonization of this HooToo HT-TM05 and the existing RAVPower RP-WD03 travel routers.

Should LED labels be named after `compatible`? E.g. `label = "ht-tm05:blue:power";` ?

It would be nice to keep the `filehub` LED label here (like in [`mt7621_elecom_wrc-gs.dtsi`](https://github.com/openwrt/openwrt/blob/d32010d5ff7455cd51fb6b3e06fa4662ad87eaa9/target/linux/ramips/dts/mt7621_elecom_wrc-gs.dtsi)) so the DTS' would differ only in `model` and `compatible`.

Reply:
Sadly/fortunately there are example for any pattern in `ramips` target:
* [`mt7628an_onion_omega2.dtsi`](https://github.com/openwrt/openwrt/blob/d32010d5ff7455cd51fb6b3e06fa4662ad87eaa9/target/linux/ramips/dts/mt7628an_onion_omega2.dtsi) - common LED definition but specific `label`s.
* [`mt7621_asus_rt-acx5p.dtsi`](https://github.com/openwrt/openwrt/blob/d32010d5ff7455cd51fb6b3e06fa4662ad87eaa9/target/linux/ramips/dts/mt7621_asus_rt-acx5p.dtsi) - kept original `label = "rt-ac85p:blue:power";` in `.dtsi` ... maybe an overlook?
* [`mt7621_elecom_wrc-gs.dtsi`](https://github.com/openwrt/openwrt/blob/d32010d5ff7455cd51fb6b3e06fa4662ad87eaa9/target/linux/ramips/dts/mt7621_elecom_wrc-gs.dtsi) - mentioned previously: a generic `label` in `.dtsi`

Reply:
In principle, the first part of the label should be the model part of the compatible. There are some exception, e.g. tp-link devices just use tp-link. Some people sneaked in other things, too.

Reply:
> mt7621_asus_rt-acx5p.dtsi - kept original label = "rt-ac85p:blue:power"; in .dtsi ... maybe an overlook?

That one should actually be fixed. Thanks for bringing this to my attention.

Reply:
Thanks for clarification!

So the [`mt7628an_onion_omega2.dtsi`](https://github.com/openwrt/openwrt/blob/d32010d5ff7455cd51fb6b3e06fa4662ad87eaa9/target/linux/ramips/dts/mt7628an_onion_omega2.dtsi) (and [`mt7628an_onion_omega2.dts`](https://github.com/openwrt/openwrt/blob/d32010d5ff7455cd51fb6b3e06fa4662ad87eaa9/target/linux/ramips/dts/mt7628an_onion_omega2.dts) + [`mt7628an_onion_omega2p.dts`](https://github.com/openwrt/openwrt/blob/d32010d5ff7455cd51fb6b3e06fa4662ad87eaa9/target/linux/ramips/dts/mt7628an_onion_omega2p.dts)) is a good example to follow?

Reply:
For hootoo and ravpower, names are totally different, so the LEDs should be "duplicated" at the respective DTS files.

Reply:
Sorry, trying to catch up a bit remotely (as I mentioned above, traveling right now). Do I need to change this back? I don't have a real opinion either way, just changed it based on input from this thread 🤣. And while I'm updating (if needed), any other items that you see, to try to get this in place?

Thanks!

Reply:
> Do I need to change this back?

Yes, please!

Reply:
And for harmonizing HooToo and RAVPower LED labels, please follow the above mentioned `mt7628an_onion_omega2.dtsi` example!

Reply:
@xabolcs Personally, I don't like this split approach, as it's hard to read. I'd just put the whole leds block into the DTSes.

With this PR still lying around, I'd personally just focus on getting this device done and wouldn't make it more complex by already taking another device into account.

Reply:
> Personally, I don't like this split approach, as it's hard to read. I'd just put the whole leds block into the DTSes.

@adrianschmutzler 
Looks like we have different preferences here, as I see more value in this split approach, but this doesn't mean anything here: the leds block should be resolved in the way you prefer!


> I don't have a real opinion either way, just changed it based on input from this thread :rofl: .

@arrmo I asked you on the forum to ask @adrianschmutzler about the led labels before changing anything!
Looks like it didn't pass through. No problem, that's why I asked here.

Reply:
I did try - never could get the registration email from the forum to show up 😢. But no biggie, will fix this.

So change the name back to "tm05" (or ht-tm05?), and separate them in the dtsi / dts files (later). Right?

Reply:
Please just let me know when you can - what name to use (as above). I can change it, NP here. Thanks!

Reply:
@arrmo see previous comment:
> In principle, the first part of the label should be the model part of the compatible.

Reply:
Agreed! But the debate above is what the model number is (i.e. tm05, or ht-tm05). I can do either quite easily 😆. Just need to know which one.

Reply:
I prefer `hootoo,ht-tm05` as compatible, so use `ht-tm05` for labels!

Reply:
yes, ht-tm05 seems to be used in marketing as well.

Reply:
Updated - thanks!